components/grid/events.md

```diff
@@ -60,6 +61,424 @@ The command buttons of a grid provide an `OnClick` event before firing their bui
 
 Fires when the item selection is enabled and the user changes the selected [item]({%slug components/grid/selection/single%}#selecteditemschanged-event) or [items]({%slug components/grid/selection/multiple%}#selecteditemschanged-event).
 
+### OnModelInit
+
+The `OnModelInit` event fires before editing and adding new item in the Grid. The event allows you to:
+
+* bind the Grid to a class that has no parameterless constructor
+
+* bind the Grid to an interface
+
+* bind the Grid to an abstract class
+
+To achieve the desired behavior you can provide an instance of the model that the Grid is bound to in the `OnModelInit` event handler.
+
```

Comment:
We could add a note that the Grid should still be bound to a single model.

Reply:
Yes, I will do it asap :)